### ISSUE TITLE
Bug in CLIP: image_fmap_size is None

### ISSUE THREAD

**Issue by jbdel** [lucidrains/DALLE-pytorch#372]
Hello,

CLIP model calls Transformer with rotary_emb=True by default, but doesnt give the image_fmap_size, this trigger an error in Transformer.

This can be replicated using the example in the readme:

```
import torch
from dalle_pytorch import CLIP

clip = CLIP(
    dim_text = 512,
    dim_image = 512,
    dim_latent = 512,
    num_text_tokens = 10000,
    text_enc_depth = 6,
    text_seq_len = 256,
    text_heads = 8,
    num_visual_tokens = 512,
    visual_enc_depth = 6,
    visual_image_size = 256,
    visual_patch_size = 32,
    visual_heads = 8
)
```

**Comment by lucidrains** [comment]
@jbdel oh yes, thanks for catching this, should be fixed in 1.0.8 https://github.com/lucidrains/DALLE-pytorch/commit/15d2f3593eb4812edd53b24753b96c3208fc4b71

**Comment by jbdel** [comment]
hooray
